js/views/Statistics.svelte

```diff
@@ -167,6 +170,14 @@
 					statUrl += `&userFilter=${value}`;
 				}
 			});
+
+			if (timezone) {
+				statUrl += `&timezone=${timezone}`;
```

Comment:
See my comment about timezones: If we send the time in UTC, we might not need to send the user's timezone along with the filter time frame

Reply:
While I changed it everywhere else, it is not possible to change it here. When grouping times based on date you need to know which timezone you group in and grouping is done on php side in `getHoursInPeriod`. Example:

You worked on 08.10.2022 23:00 - 09.10.2022 2:00 UTC, this is on 9th of October Europe/Berlin timezone, but 8th of October America/Los Angeles timezone. To be able to group that correctly, the timezone needs to be passed to the grouping function.

Actually it would be cool (and more correct), if a time entry 23:00 - 02:00 is counted splitwise with one hour in day A (23:00-00:00) and two hours in day B (00:00-02:00), but this might be something for another PR :D

Reply:
@devattendant For what it's worth, I wanted to say that you were absolutely right here. 👏  Sorry about my ignorance 😳 

The potential solution to remaining timezone issues I'm working on right now looks a bit similar. However, the [ObjectMapper](https://github.com/te-online/timemanager/blob/main/lib/Db/ObjectMapper.php#L173) needed a lot more attention, because we are sometimes querying using dates without timestamps, which caused all sorts of issues.